pkg/aws/service.go

```diff
@@ -178,37 +178,37 @@ func FromEnv(ctx context.Context) Config {
 			PrivateKey: cryptoPrivKey,
 			PublicURL:  strings.Split(mustGetEnv("PUBLIC_URL"), ","),
 			ProvidersRedis: goredis.ClusterOptions{
-				Addrs:                      []string{mustGetEnv("PROVIDERS_REDIS_URL")},
+				Addrs:                      []string{mustGetEnv("PROVIDERS_CACHE_URL")},
 				ReadOnly:                   true,
 				RouteRandomly:              true,
-				CredentialsProviderContext: redisCredentialVerifier(awsConfig, mustGetEnv("REDIS_USER_ID"), mustGetEnv("PROVIDERS_REDIS_CACHE")),
+				CredentialsProviderContext: redisCredentialVerifier(awsConfig, mustGetEnv("CACHE_USER_ID"), mustGetEnv("PROVIDERS_CACHE_ID")),
 				TLSConfig: &tls.Config{
 					MinVersion: tls.VersionTLS12,
 				},
 			},
 			NoProviderRedis: goredis.ClusterOptions{
-				Addrs:                      []string{mustGetEnv("NO_PROVIDERS_REDIS_URL")},
+				Addrs:                      []string{mustGetEnv("NO-PROVIDERS_CACHE_URL")},
 				ReadOnly:                   true,
 				RouteRandomly:              true,
-				CredentialsProviderContext: redisCredentialVerifier(awsConfig, mustGetEnv("REDIS_USER_ID"), mustGetEnv("NO_PROVIDERS_REDIS_CACHE")),
+				CredentialsProviderContext: redisCredentialVerifier(awsConfig, mustGetEnv("CACHE_USER_ID"), mustGetEnv("NO-PROVIDERS_CACHE_ID")),
```

Comment:
Should maybe be `NO_PROVIDERS_CACHE_ID` but I can't see where this is defined?

Reply:
https://github.com/storacha/storoku/blob/main/deployment/ecs_task.tf